doc/topics/releases/oxygen.rst

```diff
@@ -273,14 +273,71 @@ The new grains added are:
 * ``iscsi_iqn``: Show the iSCSI IQN name for a host
 * ``swap_total``: Show the configured swap_total for Linux, *BSD, OS X and Solaris/SunOS
 
-Salt Minion Autodiscovery
+Salt Minion Auto-discovery
 ------------------------
 
-Salt Minion now no longer need to be configured against a specifig DNS name or IP address of a Master.
+Salt Minion now no longer need to be configured against a specific DNS name or IP address of a Master.
 
 For this feature Salt Master now requires port 4520 for UDP broadcast packets to be opened
 and the Salt Minion be able to send UDP packets to the same port.
 
+Configuration
+=============
+
+By default, automatic discovery is disabled.
+
+..warning::
+   Due to the current limitations that will be changing in a future, before you turn on auto-discovery,
+   make sure your network is secured and trusted.
+
+Auto-discovery is configured on Master and Minion. Both of them are configured via ``discovery`` option
+as follows:
+
+**Master configuration**
+
+To simply enable it by default, accepting all kinds of minions, it is enough to set it to True:
+
+.. code-block:: yaml
+
+       discovery: true
+
+Sub-option `mapping` allows auto-discovery to help Minions to find a proper Master for them. The mapping contains
+just an arbitrary set of key/value strings that would match either any of them or all of them. Default is empty.
```

Comment:
This doesn't look like it was updated per my earlier review:

```
A sub-option called `mapping` allows autodiscovery to help find the proper
Master. The mapping contains an arbitrary set of key/value pairs, which the
Minion configuration can target. By default, no mappings are set.
```

Reply:
Right, ...and it is "auto-discovery". :wink: